api/src/main/java/com/sap/xsa/security/container/XSUserInfo.java

```diff
@@ -211,20 +272,27 @@
 	 * @throws XSUserInfoException
 	 *             if attribute is not available in the authentication token
 	 */
+	// TODO 14.01.20 c5295400: can this be replaced?
```

Comment:
No, have to check this with XSA
but as it is also not supported in our Spring XsuaaToken...

Reply:
Please mark with // TODO XSA?